.github/workflows/e2e-test.yml

```diff
@@ -1,15 +1,19 @@
 name: E2E Test
 
 on:
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+    branches:
+      - stage
```

Comment:
we want to be able to run it manually on any branch not just stage, why it as empty.